### DISCUSSION TITLE
RCS Enhancement

### DISCUSSION THREAD

**Discussion by kkfl** [InnovateAsterisk/Browser-Phone#455] [Ideas]
Great product, love it.  Have you considered doing some enhancements?
I think of this with potential of unified communication platform.
To that end, it would be awesome if for example we integration with What'sApp and Facebook messaging.

You just never know - some people communicate with SMS, others What'sApp, others yet with Facebook.
It sure would be nice to have one platform that allows all of this in one place.  3CX seems to have done it on their setup so I'm sure its doable.  Just no clue how much work or effort is involved.

Thanks again for a great product.

**Comment by InnovateAsterisk** [comment]
Its quite an undertaking. I’m getting there slowly. As you describe is still what I aim to achieve. Thanks for the feedback.

**Comment by kkfl** [comment]
Sounds good, let me know how I can help.  Glad to donate some programming code to the project.  Ask for a direction of code, and I'll do what I can to contribute.
